core-java/src/main/java/org/spine3/server/MultiHandler.java

```diff
@@ -36,5 +36,10 @@
     /**
      * Returns a map from methods to command classes they handle.
      */
-    Multimap<Method, Class<? extends Message>> getHandlers();
+    Multimap<Method, Class<? extends Message>> getCommandHandlers();
```

Comment:
Please revert MultiHandler to have only `getHandlers()`. Have two methods only for `ProcessManager` and treat it separately for registration.